views/forms/event/add.php

```diff
@@ -15,7 +15,7 @@
 <?php wp_nonce_field( 'acm/event/insert', 'nonce', false ); ?>
 
 <label for="event-hook"><?php esc_html_e( 'Hook', 'advanced-cron-manager' ); ?></label>
-<p class="description"><?php esc_html_e( 'Should contain only lowercase letters, numbers and _', 'advanced-cron-manager' ); ?></p>
+<p class="description"><?php esc_html_e( 'Should not contain HTML and PHP tags', 'advanced-cron-manager' ); ?></p>
```

Comment:
```suggestion
<p class="description"><?php esc_html_e( 'Should not contain HTML nor PHP tags', 'advanced-cron-manager' ); ?></p>
```

Better grammar suggestion